contrib/time_series_expectations/time_series_expectations/expectations/column_aggregate_time_series_expectation.py

```diff
@@ -0,0 +1,102 @@
+from abc import ABC
+from typing import Dict, Optional
+
+import pandas as pd
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+)
+from great_expectations.expectations.expectation import ColumnExpectation
+from time_series_expectations.expectations.util import get_prophet_model_from_json
+
+
+class ColumnAggregateTimeSeriesExpectation(ColumnExpectation, ABC):
+    """This Expectation is used to compare the values in a column to a Prophet forecast, based on a timestamp in a second column."""
+
+    # These examples will be shown in the public gallery.
+    # They will also be executed as unit tests for your Expectation.
+    # examples = []
```

Comment:
It looks like this is a valid expectation, not just an ABC - or am I missing something? If it is valid and can be used directly, let's add some examples here to use as test cases.

Reply:
It's an ABC, but the docstring was misleading. I've fixed it with a better docstring.